StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVEShopStockModifier.cs

```diff
@@ -0,0 +1,218 @@
+            const string daggerLocationName = "Tempered Galaxy Dagger";
+            var itemsData = DataLoader.Objects(Game1.content);
+            for (var i = shopData.Items.Count - 1; i >= 0; i--)
+            {
+                var item = shopData.Items[i];
+                if (!item.Id.Equals(ModItemIds.TEMPERED_DAGGER, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    continue;
+                }
+
+                var apShopItem = CreateArchipelagoLocation(item, daggerLocationName);
+                shopData.Items.RemoveAt(i);
+                shopData.Items.Insert(i, apShopItem);
+            }
+        }
+
+        private void ReplaceIsaacWeaponsWithChecks(ShopData shopData)
+        {
+            const string swordLocationName = "Tempered Galaxy Sword";
+            const string hammerLocationName = "Tempered Galaxy Hammer";
+            var itemsData = DataLoader.Objects(Game1.content);
+            for (var i = shopData.Items.Count - 1; i >= 0; i--)
+            {
+                var item = shopData.Items[i];
+                if (item.Id.Equals(ModItemIds.TEMPERED_SWORD, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    var apShopItem = CreateArchipelagoLocation(item, swordLocationName);
+                    shopData.Items.RemoveAt(i);
+                    shopData.Items.Insert(i, apShopItem);
+                }
+                else if (item.Id.Equals(ModItemIds.TEMPERED_HAMMER, StringComparison.InvariantCultureIgnoreCase))
+                {
+                    var apShopItem = CreateArchipelagoLocation(item, hammerLocationName);
+                    shopData.Items.RemoveAt(i);
+                    shopData.Items.Insert(i, apShopItem);
+                }
+            }
+        }
+
+        private void MakeBearBarter(ShopData shopData)
+        {
+            var berryItems = _stardewItemManager.GetObjectsWithPhrase("berry");
+            var newItems = new List<ShopItemData>();
+            foreach (var item in shopData.Items)
+            {
+                var random = new Random((int)Game1.stats.DaysPlayed + (int)Game1.uniqueIDForThisGame / 2 + item.GetHashCode());
+                var chosenItemGroup = berryItems.FindAll(x => !(x.Name.Contains("Joja") || x.Name.Contains("Seeds")) && x.SellPrice != 0 );
+                var chosenItem = chosenItemGroup[random.Next(chosenItemGroup.Count)];
+                var isRecipe = item.ItemId.Contains("Baked Berry Oatmeal") || item.ItemId.Contains("Flower Cookie");
+                if (isRecipe && _archipelago.SlotData.Chefsanity.HasFlag(Chefsanity.Purchases))
+                {
+                    chosenItem = BerryIfChefsanityIsOn();
+                }
+                newItems.Add(SwapItemToBerryBarter(item, chosenItem));
+            }
+            shopData.Items = newItems;
+        }
+
+        private StardewObject BerryIfChefsanityIsOn()
+        {
+            if (Game1.season.HasFlag(Season.Spring))
```

Comment:
Why the HasFlags? D:

Reply:
This is also not a Flags Enum, just an Enum. They are designed to be checked for equality.
Flags Enums, like Chefsanity, shouldn't be checked for equality, because the values are flags, not specifics

Reply:
Later comment clarified, I'll move to strict equality and rely on the previous comment to fix the issue.